rotkehlchen/history/events.py

```diff
@@ -160,11 +167,12 @@ def query_location_latest_trades(
             return
 
         for exchange in exchanges_list:
-            exchange.query_trade_history(
-                start_ts=from_ts,
-                end_ts=to_ts,
-                only_cache=False,
-            )
+            if ignored_names and exchange.name not in ignored_names:
+                exchange.query_trade_history(
+                    start_ts=from_ts,
+                    end_ts=to_ts,
+                    only_cache=False,
+                )
```

Comment:
as you have it now it doesn't make sense either. The condition is only `exchange.name not in ignored_names`